### ISSUE TITLE
"Mode selection"-window: Wording overhaul

### ISSUE THREAD

**Issue by Realchacal** [monero-project/monero-gui#2208]
![monero3](https://user-images.githubusercontent.com/46682965/59207445-5eee8500-8ba7-11e9-8be7-ab2343bec5d0.jpg)

Issues:

1.) Relating to blockchain downloads the term "bootstrap" is commonly used for fast blockchain downloads over a 3rd-party (instead over the P2P protocol). Contrary to that, in Monero-GUI the term is used to describe the temporary connection to a remote node. This is unnecessary confusing for everybody in the blockchain space.

2.) The explanations for the Simple mode and Simple mode (bootstrap) fail to explain the real difference. For example, a user has no chance to know, that Simple mode (bootstrap) will allow him to use his wallet instantly. 

3.) It is not indicated, what the EFFECTIVE consequences of the different modes are. But that's what's most important for (new!) users.

4.) In the description of the Advanced mode it says "The blockchain is downloaded to your computer" - which is wrong, since the user has full control over the corresponding settings and can choose to use a remote node.

![monero 3 vorschlag](https://user-images.githubusercontent.com/46682965/59210146-2a7dc780-8bad-11e9-97e3-c71a94276c33.jpg)

**Comment by xmrdsc** [comment]
I like the proposed changes.

**Comment by notmike-5** [comment]
#2236

**Comment by xmrdsc** [comment]
People IRC (`#monero-gui`) do not seem to like this change. Specifically; dEBRYUNE/sgp/ErCiccione. In order for this to go in, you'll have to convince multiple people - I don't have the energy for it so I'm closing my PR #2280

**Comment by notmike-5** [comment]
I have also closed #2273 so as to satisfy the Monero Politboro and Central Committee.

**Comment by xmrdsc** [comment]
That's kind of stretching it @notmike-5 ;-)

**Comment by Realchacal** [comment]
That's why I gave 4 reasons why the changes are important, so that people can argue against it (instead of having to say "they don't like it" on IRC.)

Seriously: That bootstrap-mode is ANYTHING but self-explaining. And currently it is not explained at all! Sure, veterans don't care. But for newcomers it's a slap in the face.

**Comment by dEBRUYNE-1** [comment]
@Realchacal - I was opposed to the changes mainly because it included the terms remote node and local node. New users are not acquainted with those terms and therefore they arguably should not be used. Do you have any suggestions on new wording without these terms?

**Comment by Realchacal** [comment]
@dEBRUYNE-1 fair point! We could make it this way:

- Simple mode (quick)
Basic functionality. You are connected to a third party, resulting in reduced privacy.

- Simple mode (quick + privacy)
Basic functionality. You are temporarily connected to a third party, while the blockchain is downloaded in the background. Once the download is complete, you will have maximum privacy.

- Advanced mode (maximum privacy)
Includes extra features like mining and message verification. By default, the blockchain is downloaded to your computer.

**Comment by Realchacal** [comment]
Can somebody put this into code? (I won't because I can't)

**Comment by Realchacal** [comment]
Can one of the veterans say if we're going to do this now? And then someone put it into code? (I won't cause I can't.)

**Comment by notmike-5** [comment]
@Realchacal I think they already said no. PR #2273 has the code you're looking for.

**Comment by selsta** [comment]
No we didn’t say no, else the issue would be closed.

**Comment by Realchacal** [comment]
I'd put this issue on hold because it would be superseded if #2320 was adopted.
